data_import/models.py

```diff
@@ -81,29 +69,11 @@ class DataFileManager(models.Manager):
     pre_delete signal connected correctly.
     """
 
-    def for_user(self, user):
-        return (
-            self.filter(user=user)
-            .exclude(parent_project_data_file__completed=False)
-            .order_by("source")
-        )
-
     def contribute_to_class(self, model, name):
         super(DataFileManager, self).contribute_to_class(model, name)
 
         models.signals.pre_delete.connect(delete_file, model)
 
-    def public(self):
-        prefix = "user__member__public_data_participant__publicdataaccess"
-
-        filters = {prefix + "__is_public": True, prefix + "__data_source": F("source")}
```

Comment:
This was moved to ProjectDataFile and switched away from the "data source" approach (as this was removed from PublicDataAccess).